packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue

```diff
@@ -49,5 +49,10 @@ export default {
     &:last-of-type {
         margin-right: 0;
     }
+
+    &:before {
```

Comment:
Is this an issue specific to searchweb? If so, should it live in search as apposed to this component (if its tech debt).

Reply:
in theory it's an issue for any legacy sites in the same position as searchweb using similar SCSS configuration @sonnyprince - happy to move it if you'd prefer

Reply:
shall do!